### PR TITLE
added ok-computer

### PR DESCRIPTION
**Pull request by richardscarrott**
https://github.com/moltar/typescript-runtime-type-benchmarks/issues/696

### CODE REVIEW

**Comment by richardscarrott** [comment]
Not sure why ok-computer is so slow as, although I haven't / don't intent to make micro optimisations, it's running a few orders of magnitude slower than others; I'll push a new version once I've done some optimisation.

**Comment by moltar** [comment]
@richardscarrott See #864, which might affect the benchmarks.

@hoeck has a PR for this in #866, which will get out soon, so might want to wait for that and retry your benchmarks.

Cheers and thanks for this PR!

File: package.json [hunk 1]
```diff
@@ -43,6 +43,7 @@
     "jointz": "6.1.1",
     "json-decoder": "1.4.0",
     "myzod": "1.8.6",
+    "ok-computer": "^1.0.0",
```
Comment (moltar) [inline package.json:46]:
```suggestion
    "ok-computer": "1.0.0",
```

**Review by moltar — CHANGES_REQUESTED** [review]
One small change though. We pin the exact version to make sure that we record the bench results against the exact version.

File: package-lock.json [hunk 2]
```diff
@@ -32,6 +32,7 @@
         "jointz": "6.1.1",
         "json-decoder": "1.4.0",
         "myzod": "1.8.6",
+        "ok-computer": "^1.0.0",
```
Comment (moltar) [inline package-lock.json:35]:
```suggestion
        "ok-computer": "1.0.0",
```

**Review by moltar — APPROVED** [review]

**Comment by richardscarrott** [comment]
@moltar I did some profiling this morning and found it was genuinely slower than it should have been due to hitting a catch block frequently. I made a fix here https://github.com/richardscarrott/ok-computer/pull/7 and sent an update to the benchmarks here https://github.com/moltar/typescript-runtime-type-benchmarks/pull/869

**Comment by moltar** [comment]
Awesome!!

**Pull request merged**